main.js

```diff
@@ -5152,9 +5152,17 @@ function addProfile(profession, profile, base){
                 limit: 0
             };
             _vendorItems[_vendorItems.length] = {
-                pattern: /_Green_T[1-5]_Unid$/,
+                pattern: /_Green_T[1-7]_Unid$/,

```

Comment:
@noonereally what do you think about this?

Before we sold up to Green_T5 so people gathering greens (Green_T6) for refining were safe to do so.

I guess we should keep that way and now sell only to Green_T6 so lvl 70 greens can still be kept in the inventory.